### PR TITLE
update shebang line of all scripts to be more compatible between *nix environments

### PR DESCRIPTION
**Pull request by alice-wondered**
On OSX specifically, the OS ships with a very old version of bash that's missing many features used in various scripts in this project. Installing from brew allows for the env bin to point to a much newer version of bash and resolves this issue

### CODE REVIEW

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for fixing this. Could you please add the DCO to your commit like we do in the Keycloak main repository? Once that is in, it is good for merge.

**Comment by kami619** [comment]
@ahus1 https://github.com/keycloak/keycloak-benchmark/actions/runs/7453510299/job/20328347373?pr=672#step:4:5 Please note that the Documentation CI job failed with the recent change from @alice-wondered

File: antora/author.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
-#!/bin/bash -e
+#!/usr/bin/env bash -e
```
Comment (ahus1) [inline antora/author.sh:1]: With the GitHub action failing, and when looking at it in my IDE (IntelliJ) which Shell Check enabled, it provided the following link that explains the problem and the remedy: 

https://github.com/koalaman/shellcheck/wiki/SC2096

Please update all occurrences like the one below.

```suggestion
#!/usr/bin/env bash 
set -e
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Apparently there was a problem lurking in this - could you please have a look and fix the scripts, so they work well with the parameter? Thanks!

File: antora/author.sh (hunk 1 above)
Comment (alice-wondered) [reply antora/author.sh:1]: No problem, I'll run through and correct these

Comment (alice-wondered) [reply antora/author.sh]: Sorry about the oversight! Should be resolved now (and I also picked up that shell check plugin 😄)

**Review by ahus1 — APPROVED** [review]
Thank you for this change!

**Pull request merged**